mkt/websites/management/commands/import_games_from_csv.py

```diff
@@ -117,7 +117,7 @@ def create_instances(self, data):
                     # Keywords use a M2M, so do that once the website is saved.
                     self.set_tags(website, row)
 
-                    WebsiteIndexer.index([website.id], no_delay=True)
+                    WebsiteIndexer.index_ids([website.id], no_delay=True)
```

Comment:
We have it, I added it back during Feed. https://github.com/mozilla/zamboni/blob/master/mkt/search/indexers.py#L99